### PR TITLE
Cms default page

### PR DESCRIPTION
**Pull request by apradillap**
Connects to #925 

### What does this PR do?
http://madrid.gobierto.dev/sandbox/cms_default_page.html

### How should this be manually tested?
You can test this functionality by creating a site type collection and accessing the page after creating it.

### CODE REVIEW

**Comment by apradillap** [comment]
@furilo I have some questions:
- In the main breadcrumb will display the name of the collection? Nowadays, only pages can be associated with a site, process or people
- In the breadcrumb: "Transparencia / Publicidad Activa / Información sobre la corporación municipal" In our case we'll have the name of the collection. Should it link somewhere?

**Comment by furilo** [comment]
For the time being, as we don't have hierarchy, let's show just the collection name as first and only element of the breadcrumb. We'll iterate in further issues.

**Comment by apradillap** [comment]
@furilo, you can test the first approach in http://getafe.gobify.net/paginas/wadus

**Comment by apradillap** [comment]
@furilo,
- I changed the slug from admin cms link
- Hide process menu subsections but main_nav is very high

You can test it in participation

**Comment by furilo** [comment]
- [ ] The sidebar menu is not linking correctly to pages
- [ ] Neither the bredcrumb. Actually, let's remove for the moment the
links for the breadcrumb

**Comment by apradillap** [comment]
Fixed your comments @furilo

**Comment by furilo** [comment]
Just one bit: - [ ] We are not adding a proper meta title tag

Then, merge and deploy on Dival to check.

**Comment by furilo** [comment]
(or ask @amiedes to deploy, who has another deploy pending)

File: app/models/gobierto_common/collection.rb [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
     has_many :collection_items, dependent: :destroy
 
     translates :title
 
-    validates :site, :title, :slug, :item_type, presence: true
+    validates :site, :title, :item_type, presence: true
     validates :container, presence: true, associated: true
     validates_associated :container
-    validates :slug, uniqueness: true
+    validates :slug, uniqueness: { scope: :site }
```
Comment (ferblape) [inline app/models/gobierto_common/collection.rb:15]: 👍

**Review by ferblape — APPROVED** [review]
👍

**Comment by ferblape** [comment]
Deployed to Dival!

**Pull request merged**